wgpu-types/src/lib.rs

```diff
@@ -1026,6 +1026,13 @@ bitflags::bitflags! {
         ///
         /// GLES/WebGL don't support this.
         const DEPTH_TEXTURE_AND_BUFFER_COPIES = 1 << 13;
+
+        /// Supports rendering to floating-point textures.
+        const COLOR_ATTACHMENT_FLOAT = 1 << 14;
```

Comment:
We should have this feature at all, as you can just query support for the texture you need when WEBGPU_TEXTURE_FORMAT_SUPPORT isn't available.